server/core/provider.go

```diff
@@ -13,6 +13,8 @@ type (
 		Name        string     `gorm:"not null" json:"name"`
 		URL         string     `gorm:"not null" json:"url"`
 		AccessToken string     `gorm:"not null" json:"-"`
+		HttpUser    string     `json:"HttpUser"`
+		HttpPass    string     `json:"HttpPass"`
```

Comment:
You should not store the password as a plain text. 

The json tag names should start with lowercase https://jsonapi.org/recommendations/